### PR TITLE
test(Time Lock): Change function call to work with solidity coverage

### PR DESCRIPTION
**Pull request by tcrosado**
## AppCoins Protocol smart contracts

Fix function call of getBalanceOf function to work with solidity coverage

Before opening a pull request, please ensure:

- [x] You have followed our [**contributing guidelines**](https://github.com/AppStoreFoundation/asf-contracts/wiki/Contributing-to-AppCoins-Protocol-smart-contracts)
- [x] double-check your branch is based on `dev` and targets `dev` 
- [x] Pull request has tests (at least 80% coverage)
- [x] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Description explains the issue/use-case resolved and auto-closes related issues

### CODE REVIEW

File: test/appCoinsTimelock.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ contract('AppCoinsTimelock', function(accounts) {
         await AppCoinsTimelockInstance.allocateFunds(testAddresses[0], testAmounts[0]);
```
Comment (DioPires) [inline test/appCoinsTimelock.js:40]: #### Tests:
```diff 
+ 104 passing
```
#### Test coverage: 
81.44 %

Comment (DioPires) [inline test/appCoinsTimelock.js:40]: #### Tests:
```diff 
+ 104 passing
```
#### Test coverage: 
81.44 %

Comment (DioPires) [inline test/appCoinsTimelock.js:40]: #### Tests:
```diff 
+ 104 passing
```
#### Test coverage: 
81.44 %

Comment (DioPires) [inline test/appCoinsTimelock.js:40]: #### Tests:
```diff 
+ 104 passing
```
#### Test coverage: 
81.44 %

**Review by GustavoRSSilva — APPROVED** [review]

**Pull request merged**